### ISSUE TITLE
bootctl prints ANSI escape codes with TERM=dumb

### ISSUE THREAD

**Issue by dblsaiko** [systemd/systemd#31857]
### systemd version the issue has been seen with

systemd-254.6

### Used distribution

NixOS 23.11

### Linux kernel version used

6.1.80

### CPU architectures issue was seen on

x86_64

### Component

bootctl

### Expected behaviour you didn't see

`TERM=dumb bootctl` does not include ANSI escape codes

### Unexpected behaviour you saw

`TERM=dumb bootctl` includes ANSI escape codes such as ^[0m

### Steps to reproduce the problem

Run `TERM=dumb bootctl | xxd`

### Additional program output to the terminal or log subsystem illustrating the issue

```sh
# TERM=dumb bootctl | xxd | head -1
00000000: 1b5b 306d 5379 7374 656d 3a0a 2020 2020  .[0mSystem:.
```

**Comment by rpigott** [comment]
You can disable color with SYSTEMD_COLOR=0.

**Comment by 2xsaiko** [comment]
As an environment variable? It doesn't seem to work for bootctl. At least, I've tried SYSTEMD_COLOR=0 bootctl.

**Comment by yuwata** [comment]
Fix is waiting in #31859.
